pkg/drift/drift.go

```diff
@@ -175,3 +217,26 @@ func URI(i *iam.AssetIAM, organizationID string) string {
 		return fmt.Sprintf("/organizations/%s/%s/%s", organizationID, role, i.Member)
 	}
 }
+
+// driftignore parses the driftignore file into a set.
+// Go doesn't implement set so we use a map of boolean values all set to true.
+func driftignore(ctx context.Context, fname string) (map[string]bool, error) {
+	lines := make(map[string]bool)
+	f, err := os.Open(fname)
+	if err != nil {
+		if os.IsNotExist(err) {
+			logger := logging.FromContext(ctx)
+			logger.Debugw("failed to find driftignore", "filename", fname)
+			return lines, nil
+		}
+		return nil, fmt.Errorf("failed to read driftignore file %s: %w", fname, err)
+	}
+	defer f.Close()
+
+	scanner := bufio.NewScanner(f)
+	for scanner.Scan() {
+		lines[scanner.Text()] = true
```

Comment:
I would probably `strings.TrimSpace` the results and ignore any empty things.

Reply:
Done